src/Configurator.php

```diff
@@ -63,6 +68,15 @@ private function __construct()
         $this->config = new ApplicationConfig([]);
     }
 
+    public static function container()
```

Comment:
Should we add a docblock here to make it transparent that this method returns a `string`?